api/models/photo.py

```diff
@@ -836,6 +838,19 @@ def _get_dominant_color(self, palette_size=16):
         except Exception:
             logger.info("Cannot calculate dominant color {} object".format(self))
 
+    def _get_blurhash(self, palette_size=16):
+        # Skip if it's already calculated
+        if self.blurhash:
+            return
+        try:
+            hash = blurhash.encode(
+                self.square_thumbnail_small.path, x_components=4, y_components=4
```

Comment:
square_thumbnail_small becomes redundant once thumbhash or blurhash is fully implemented, so use square_thumbnail instead.
Currently, we create white placeholders on the frontend, then update their color to match the dominant color upon receiving a paged response. Next, we use square_thumbnail_small, apply a blur effect, and display it as a temporary preview. Finally, the actual thumbnail is loaded for the timeline. Once thumbhash or blurhash is implemented, dominant color and square_thumbnail_small are redundant and will probably be removed in the future :)